integration/test/performancetest/resources/config.json

```diff
@@ -29,17 +59,56 @@
 				"measurement": [
 					"mem_used_percent"
 				],
-				"metrics_collection_interval": 60
+				"metrics_collection_interval": 1
+			},
+			"netstat": {
+				"measurement": [
+					"tcp_established",
+					"tcp_time_wait"
+				],
+				"metrics_collection_interval": 1
+			},
+			"procstat": [
+                		{
+                    			"exe": "cloudwatch-agent",
+                    			"measurement": [
+                        			"cpu_usage",
+						"memory_rss"
+                    			],
+					"metrics_collection_interval": 10
+                		}
+            		],
+			"statsd": {
+				"metrics_aggregation_interval": 10,
+				"metrics_collection_interval": 10,
+				"service_address": ":8125"
 			},
-            "procstat": [
-                {
-                    "exe": "cloudwatch-agent",
-                    "measurement": [
-                        "cpu_usage",
-			"memory_rss"
-                    ]
-                }
-            ]
+			"swap": {
+				"measurement": [
+					"swap_used_percent"
+				],
+				"metrics_collection_interval": 1
+			}
+		}
+	},
+	"logs": {
+		"logs_collected": {
+		  "files": {
+			"collect_list": [
+			  {
+				"file_path": "/tmp/test1.log",
+				"log_group_name": "{instance_id}",
+				"log_stream_name": "{instance_id}",
+				"timezone": "UTC"
+			  },
+			  {
+				"file_path": "/tmp/test2.log",
+				"log_group_name": "{instance_id}",
+				"log_stream_name": "{instance_id}",
```

Comment:
I would refrain from publishing both of them to the same log stream cause it runs the risk of hitting a rate limit. I would add some arbitrary string to differentiate them, like 
```
log_stream_name: "{instance_id}/tmp2"
```